test/functional/fixtures/regression/gh-7377/testcafe-fixtures/index.js

```diff
@@ -0,0 +1,13 @@
+import { Selector } from 'testcafe';
+
+fixture `fixture`
+    .page `http://localhost:3000/fixtures/regression/gh-7377/pages/index.html`;
+
+const input8 = Selector('#input8').find('input');
+
+
+test('test', async (t) => {
+    await t.resizeWindow(500, 270);
```

Comment:
we can remove `resizeWindow` from here. You can make a spacer with great height value. After this spacer you can put  the target element